MoltenVK/MoltenVK/Utility/MVKFoundation.h

```diff
@@ -475,6 +475,13 @@ bool contains(const C& container, const T& val) {
 	return false;
 }
 
+/** Returns whether the container contains an item equal to the value. */
+template<class C, class T>
+bool contains(C& container, const T& val) {
+	for (const T& cVal : container) { if (cVal == val) { return true; } }
```

Comment:
So this is a repeat of the `const` version.

Can we delete the `const` version? Or more to the point, just remove the `const` declaration on it?

I probably just brain-farted on that one. None of the other container iterator templates declare the container as `const`. And in any case, it doesn't look to be used by current code anyway.

Reply:
I actually wanted to use the `const` version, but I couldn't make it work with the pointer specialization of `MVKSmallVector`. Done.